config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_cloudbuildworkerpools.cloudbuild.cnrm.cloud.google.com.yaml

```diff
@@ -164,6 +164,10 @@ spec:
                       type: string
                   type: object
                 type: array
+              externalRef:
+                description: The GCP URL to the cloudbuild workerpool resource. The
```

Comment:
Maybe "A unique specifier for the CloudBuild workerpool resource in GCP."

I don't think you need to say "The cloudbuild workerpool does not have selfLink.".  (That might have been a code comment, it got pulled in here though)

Reply:
Sounds good!